packages/flutter/lib/src/rendering/viewport_offset.dart

```diff
@@ -150,10 +150,21 @@ abstract class ViewportOffset extends ChangeNotifier {
   /// [RenderViewport], before [applyContentDimensions]. After this method is
   /// called, the layout will be recomputed and that may result in this method
   /// being called again, though this should be very rare.
+  ///
+  /// See also:
+  ///
+  ///  * [jumpTo], for also changing the scroll position but applying the
+  ///    change immediately and notifying its listeners.
```

Comment:
mention that jumpTo is for when you're not in layout.

Reply:
done